cmd/openshift-tuned.go

```diff
@@ -42,12 +42,18 @@ type sockAccepted struct {
 	err  error
 }
 
+type resync struct {
+	now int64
```

Comment:
I see your C coming through using int64 for time rather than `time` objects.

Reply:
It is unlikely `time` object is needed for what this program is trying to do and I (perhaps wrongly) prefer simpler data types.  All that's needed is seconds from epoch.  If the variable naming is a problem, can rename to something like `secFromEpoch`, but `now` is just shorter...